### DISCUSSION TITLE
Rigtools change request

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1307] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So after I get on the rigtools page to put in the extension Id it opens the page and instantly get blocked by goguardian what do I do and gogudarin is the only extension with unsafe eval  can rigtools open a different link instead of system:chrome-extension://gndmhdcefbhlchkhipcnnbkcmicncehk/temporary/index.html  I want it to be unblocked so I can do the swamp on goguardian script to soft disable  my goguardian

**Comment by akabutnicer** [comment]
yeah so we can't just do that 😐

**Comment by schoolexploitkid** [comment, marked as answer]
just use disable injected extension quickly

**Comment by OPcoolman** [comment]
bro just go to history and get the rigtools lol
